src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common/interpolate.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/interpolate.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "openvino/core/preprocess/pre_post_process.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov::test::Interpolate {
+namespace {
+
+// pillow modes: planar layout with axis[1,2] executed as nhwc layout case
+const std::vector<std::vector<int64_t>> defaultAxes4D_pillow_nchw_as_nhwc = {
+        {1, 2}
+};
```

Comment:
```suggestion
const std::vector<int64_t> defaultAxes4D_pillow_nchw_as_nhwc = {
        {1, 2}
};
```
Always only the first element is accessed. It's better to use only one vector space dimention: